docusaurus.config.js

```diff
@@ -216,6 +216,8 @@ module.exports = {
           //cacheTime: 600 * 1000, // 600 sec - cache purge period
           changefreq: "weekly",
           priority: 0.5,
+          // this is temporary. Ignore these from the sitemap while we move a lot of things around.
+          ignorePatterns: ["/docs/next/**", "/optimize/**"],
```

Comment:
There are no `/optimize/**` matches in this branch, currently....but I'll be rebasing #1166 on main after this is merged, and there are matches there.